ios/MixpanelReactNative.swift

```diff
@@ -131,11 +131,12 @@ open class MixpanelReactNative: NSObject {
      
     @objc
     func identify(_ token: String, distinctId: String,
-                  resolver resolve: RCTPromiseResolveBlock,
-                  rejecter reject: RCTPromiseRejectBlock) -> Void {
+                  resolver resolve: @escaping RCTPromiseResolveBlock,
+                  rejecter reject: @escaping RCTPromiseRejectBlock) -> Void {
```

Comment:
Do we also need to update the Android side?

Reply:
android is a sync call so it should be fine